### ISSUE TITLE
Resolve Links from mycloud.to

### ISSUE THREAD

**Issue by Prometheusx-git** [tknorris/script.module.urlresolver#350]
Hi,
this is rather a wish than an issue.
Could you ad support to resolve links from mycloud.to ?

Example Link :

Thanks in advance.

**Comment by Viper2k4** [comment]
Could not test it right now but could the try this:

creat an file mycloud.py and add this:

```
from __generic_resolver__ import GenericResolver

class MycloudResolver(GenericResolver):
    name = "mycloud"
    domains = ['mycloud.to']
```

and the copy it to **_lib/urlresolver/plugins_** not the links should work. 

I will test it later.

**Comment by Prometheusx-git** [comment]
Unfortunately it is not that simple.
I've tried your approach on different links, it always leads to a HTTP 404.


```
"/storage/.kodi/addons/script.module.urlresolver/lib/urlresolver/lib/net.py", line 279, in _fetch
                                                response = urllib2.urlopen(req)                                          

 "/usr/lib/python2.7/urllib2.py", line 154, in urlopen
                                              File "/usr/lib/python2.7/urllib2.py", line 435, in open
                                              File "/usr/lib/python2.7/urllib2.py", line 548, in http_response
                                              File "/usr/lib/python2.7/urllib2.py", line 473, in error
                                              File "/usr/lib/python2.7/urllib2.py", line 407, in _call_chain
                                              File "/usr/lib/python2.7/urllib2.py", line 556, in http_error_default
                                            HTTPError: HTTP Error 404: Not Found
```

**Comment by Viper2k4** [comment]
it is im just stupid ^^

```
from __generic_resolver__ import GenericResolver

class MycloudResolver(GenericResolver):
    name = "mycloud"
    domains = ['mycloud.to']

def get_url(self, host, media_id):
    return self._default_get_url(host, media_id, template='http://{host}/embed/{media_id}')
```

**Comment by Prometheusx-git** [comment]
Sorry ,even with get_url it doesn't work.

```
10:02:38.675 T:545584701664   ERROR: URLResolver: HTTPError Error - From: mycloud Link: http://mycloud.to/embed/e4lnnj: HTTP Error 404: Not Found
10:02:38.777 T:545584701664   ERROR: Previous line repeats 1 times.
10:02:38.777 T:545584701664   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                             - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                            Error Type: <class 'urllib2.HTTPError'>
                                            Error Contents: HTTP Error 404: Not Found
                                            Traceback (most recent call last):
                                              File "/storage/.kodi/addons/plugin.video.unofficial9anime/default.py", line 66, in <module>
                                                control_table[args.action](controller.Controller())
                                              File "/storage/.kodi/addons/plugin.video.unofficial9anime/default.py", line 53, in <lambda>
                                                'qualityPlayer':     lambda c: c.choose_quality_play_video(),
                                              File "/storage/.kodi/addons/plugin.video.unofficial9anime/resources/lib/common/controller.py", line 122, in choose_quality_play_video
                                                qp.play()
                                              File "/storage/.kodi/addons/plugin.video.unofficial9anime/resources/lib/players/videoplayer.py", line 59, in play
                                                url = urlresolver.resolve(self.link)
                                              File "/storage/.kodi/addons/script.module.urlresolver/lib/urlresolver/__init__.py", line 123, in resolve
                                                return source.resolve()
                                              File "/storage/.kodi/addons/script.module.urlresolver/lib/urlresolver/hmf.py", line 180, in resolve
                                                stream_url = resolver.get_media_url(self._host, self._media_id)
                                              File "/storage/.kodi/addons/script.module.urlresolver/lib/urlresolver/plugins/__generic_resolver__.py", line 49, in get_media_url
                                                return helpers.get_media_url(self.get_url(host, media_id)).replace(' ', '%20')
                                              File "/storage/.kodi/addons/script.module.urlresolver/lib/urlresolver/plugins/lib/helpers.py", line 180, in get_media_url
                                                response = net.http_GET(url, headers=headers)
                                              File "/storage/.kodi/addons/script.module.urlresolver/lib/urlresolver/lib/net.py", line 198, in http_GET
                                                return self._fetch(url, headers=headers, compression=compression)
                                              File "/storage/.kodi/addons/script.module.urlresolver/lib/urlresolver/lib/net.py", line 279, in _fetch
                                                response = urllib2.urlopen(req)
                                              File "/usr/lib/python2.7/urllib2.py", line 154, in urlopen
                                              File "/usr/lib/python2.7/urllib2.py", line 435, in open
                                              File "/usr/lib/python2.7/urllib2.py", line 548, in http_response
                                              File "/usr/lib/python2.7/urllib2.py", line 473, in error
                                              File "/usr/lib/python2.7/urllib2.py", line 407, in _call_chain
                                              File "/usr/lib/python2.7/urllib2.py", line 556, in http_error_default
                                            HTTPError: HTTP Error 404: Not Found
                                            -->End of Python script error report<--
```

**Comment by Viper2k4** [comment]
Mh ok then i will take a look. But at all the Site is easy. Because if u take a look in the source of the link u will see the link direct in the sources.

**Comment by Prometheusx-git** [comment]
yes you are right. Thank you.

**Comment by tknorris** [comment]
Resolved with #351
